app/views/govuk_publishing_components/components/_step_by_step_nav_related.html.erb

```diff
@@ -3,7 +3,9 @@
   pretitle ||= t("govuk_component.step_by_step_nav_related.part_of", default: "Part of")
 %>
 <% if links.any? %>
-  <div class="gem-c-step-nav-related" data-module="track-click">
+  <div 
+    class="gem-c-step-nav-related <% if links.length == 1 %> gem-c-step-nav-related-singular <% end %>" 
```

Comment:
Could be written more cleanly as

```
class="gem-c-step-nav-related <%= "gem-c-step-nav-related-singular" if links.length == 1 %>" 
```

Reply:
Thanks I will update this, we'll need one as we only want it to match if length is 1